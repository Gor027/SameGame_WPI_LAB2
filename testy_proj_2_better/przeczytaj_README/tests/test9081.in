 AAAH
A AHAAAHH HAA  AHH HA A A AHAHHAAHHA  HAHAAHH  AHA H  HAH
AHAHHAAAA   AHHAHA HHA A HHHH A HHHHHH  AHAA  AH A 
HHHA   A    A HHHA HHAHA H  HHA HH HAHH AA
AA HH H H H   A H   AAHHAA  A AAAAHH AA HH
HH  HAHHHHA  AAA AHA HHA 
H  AHA   H   HH    AAHAHHA HH  AHHHH  HAAAA   HH AHA HH  
HAAHAAAA   AAAHH A AAHA H AA HAHHA  AA AA A   AHA
HHHH  AA AH HA A AH AHAHA HAHHAHHH HAAHHA HH A HAHAH A  AA    
AAAHHHHAHAHAAHAH HHAA A A
H HH      AH
