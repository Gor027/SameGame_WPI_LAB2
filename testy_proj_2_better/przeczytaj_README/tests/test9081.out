   A
A  H
AAAHH               A A
HHAAA AH      HA H HHAH         AH      H
AHAHHAAA  H AHHA A HHAHAA   A H HH A    A
HAHAHAHAH H AAAH A HAAHAA  HH AAHH H HH AAA    A
HHAHHAHHAHAAAHAAHA AAHAHHHAHHHAAAAHHHHHAHHA    H A
HAHHHAAAHHHAAAHHHAHAAHAAHHHHAHAAHHHHAAAAAAAH  AHHA H
HHAHAAAAHAHAHAHAHAHAAHAHAAAAHHAHHAHHAAHAAHAA  HHAAHAHH H
AAHHHHHAHAAHAHAHHHHAAHAHAAHAHHAHHHHHAAHHAAHHHAAHAHAHAHAAA
